modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItFunctionsTest.java

```diff
@@ -68,6 +68,13 @@ public void testLength() {
         assertQuery("SELECT LENGTH(NULL)").returns(NULL_RESULT).check();
     }
 
+    @Test
+    public void testOctetLength() {
+        assertQuery("SELECT OCTET_LENGTH('TEST')").returns(4).check();
```

Comment:
let's also add different tests for different languages to make sure OCTET_LENGTH differs from regular LENGTH